src/Microsoft.VisualStudio.InteractiveWindow.Tests/TestUtilities/Threading/ForegroundThreadAffinitizedObject.cs

```diff
@@ -33,12 +28,7 @@ internal static ForegroundThreadData CreateDefault(ForegroundThreadDataKind defa
         {
             var kind = ForegroundThreadDataInfo.CreateDefault(defaultKind);
 
-            // None of the work posted to the foregroundTaskScheduler should block pending keyboard/mouse input from the user.
-            // So instead of using the default priority which is above user input, we use Background priority which is 1 level
-            // below user input.
-            var taskScheduler = new SynchronizationContextTaskScheduler(new DispatcherSynchronizationContext(Dispatcher.CurrentDispatcher, DispatcherPriority.Background));
```

Comment:
why this change?

Reply:
Removed this change and used a bunch of suppressions instead.